bin/dipy_det_track

```diff
@@ -0,0 +1,9 @@
+#!python
```

Comment:
Looks like some of these have `/usr/bin/env python` in the shebang, and others have just `python`. Which one should it be?

Reply:
It should be just python. Corrected.

Reply:
I know we discussed about that a long time ago, but using a straight `python` doesn't work when using a virtualenv. Using the `/usr/bin/env python` correctly works. Is there a case where `/usr/bin/env` doesn't select the correct python?

Reply:
Not sure what you mean.

Reply:
I mean that if the shebang is only `#!python`, calling this script doesn't work correctly when working from inside a virtualenv. If the shebang is changed to `/usr/bin/env python`, it works correctly.